_episodes/02-variables.md

```diff
@@ -339,15 +342,15 @@ print(ewr_422_yY, 'is', flabadab, 'years old')
 > 3.  What does `thing[:high]` (without a value before the colon) do?
```

Comment:
This PR will have some conflicts with https://github.com/swcarpentry/python-novice-gapminder/pull/567/files which I'd like to merge in as well - perhaps we can revisit it after that is merged?